# scrnaseq-resources
A list of resources on scRNA-seq analysis including tutorials and guides. They are split broadly into 'processing' and 'analysis' categories, where 'processing' describes the process from raw data to expression matrices, and 'analysis' describes the subsequent processes to understand and visualise the data.

## Processing Tutorials
 - CellRanger from 10X Genomics: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorial_ov
 
 - Kallisto/bustools:
   - introduction: https://www.kallistobus.tools/introduction
   - tutorial: https://www.kallistobus.tools/tutorials (python and R)

## Analysis Tutorials
### R
 - Seurat vignette/tutorial from the Satija Lab (who developed Seurat): https://satijalab.org/seurat/v3.1/pbmc3k_tutorial.html

### Python
 - "Bioinformatics for Benched Biologists" Youtube-recorded Zoom stream from Prof. Lior Pachter and colleagues: https://www.youtube.com/watch?v=vKsh_ovq8x8 - includes a Google Cloud link to an interactive notebook so you can code along with them and use Google's servers for computation
 
 ## Complete courses:
 - Sanger Institute scRNA-seq two day course: https://scrnaseq-course.cog.sanger.ac.uk/website/index.html
 - Broad Institute: https://broadinstitute.github.io/2019_scWorkshop/ - CellRanger, Seurat, and others


## Other
 - Dizzyingly-long list of every scRNA-seq software available: https://github.com/seandavi/awesome-single-cell (n.b includes a list of tutorials)
 
 
 
 
 
 
 
